config/settings.yml

```diff
@@ -273,6 +273,11 @@ vet360:
     api_key: "<AV_KEY>"
   health_benefit:
     mock: true
+  profile_information:
+    cache_enabled: true
```

Comment:
Where is cache_enabled used?

Reply:
When cacheing redis. You can see it throughout,  VAProfile::Configuration::SETTINGS.profile_infromation.cache_enabled

Reply:
The setting will be need to be reformatted in the future